jobs/build_docker/prepare_docker_post_test.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash 
+set -x
+# trap "echo $SUDO_PASSWORD |sudo -S chown -R $USER:$USER $WORKSPACE/RackHD" SIGINT SIGTERM SIGKILL EXIT
+
+# Stop host mongo and rabbitmq service
+# otherwise mongo and rabbitmq service won't run normally
+# use host mongo and rabbitmq is a chioce but the coverage may be decreased.
+# services will be restart in cleanup.sh, which script will always be executed.
+echo $SUDO_PASSWORD |sudo -S service mongodb stop
+echo $SUDO_PASSWORD |sudo -S service rabbitmq-server stop
+
+rackhd_docker_images=`ls ${DOCKER_PATH}`
+build_record=`ls ${DOCKER_RECORD_PATH}`
+
+# load docker images
+docker load -i $rackhd_docker_images
+image_list=`head -n 1 $build_record`
+
+cd $WORKSPACE/RackHD/docker
+# replace default config json with the one which is for test.
+cp -f ${WORKSPACE}/build-config/vagrant/config/docker/config.json ./monorail/config.json
+#if clone file name is not repo name, this scirpt should be edited.
+for repo_tag in $image_list; do
+    repo=${repo_tag%:*}
+    sed -i "s#${repo}.*#${repo_tag}#g" docker-compose-mini.yml
+done
+
+mkdir -p $WORKSPACE/build-deps
+docker-compose -f docker-compose-mini.yml up > $WORKSPACE/build-deps/vagrant.log &
```

Comment:
thank you, talking with @changev , docker-compose up with "-d" can run on background, but the log will be lost